scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml

```diff
@@ -1,3 +1,68 @@
+LFCSP-WD-8-1EP_3x3mm_P0.65mm_EP1.6x2.44mm:
+  device_type: 'LFCSP-WD'
+  library: Package_CSP
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/lfcspcp/CP_8_19.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 2.9
+    nominal: 3
+    maximum: 3.1
+  body_size_y:
+    minimum: 2.9
+    nominal: 3
+    maximum: 3.1
+  body_height:
+    minimum: 0.7
+    nominal: 0.75
+    maximum: 0.8
+
+  lead_width:
+    maximum: 0.35
+    nominal: 0.30
+    minimum: 0.20
+  lead_len:
+    minimum: 0.35
+    nominal: 0.4
+    maximum: 0.45
+
+  EP_size_x:
+    maximum: 1.70
+    nominal: 1.60
+    minimum: 1.50
+  EP_size_y:
+    maximum: 2.54
+    nominal: 2.44
+    minimum: 2.34
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 3]
+  #heel_reduction: 0.05 #for relatively large EP pads (increase clearance)
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.3
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
```

Comment:
Unnecessary if paste_avoid_via is false.

Reply:
done